src/backend/PortabilityService.Gateway/appsettings.json

```diff
@@ -0,0 +1,269 @@
+﻿{
+  "ApplicationInsights": {
+    "InstrumentationKey": "<AI-Key-Goes-Here>"
+  },
+  "Gateway": {
```

Comment:
I'm debating moving the Gateway/ReRoutes config to its own file since it is so large. I'm not sure if that would be helpful or not.

Reply:
I like the idea of a separate `appsettings.gateway.json` due to the size

Reply:
Sounds good. I'll move it.